geostructures/structures.py

```diff
@@ -1320,11 +1342,21 @@ def __init__(  # pylint: disable=too-many-arguments
     ):
         super().__init__(holes=holes, dt=dt, properties=properties)
         self.center = center
-        self.inner_radius = inner_radius
-        self.outer_radius = outer_radius
+        try:
+            self.inner_radius = float(inner_radius)
+            self.outer_radius = float(outer_radius)
+        except ValueError:
+            raise ValueError("radius distance should be in meters")
         self.angle_min = angle_min
         self.angle_max = angle_max
 
+        if not isinstance(center, Coordinate):
+            raise Exception('The center must be a coordinate. The input value was: {}'.format(center))
+        if angle_min < 0 or angle_min > 360:
```

Comment:
Why does it have to be bound to 0 <= angle_min <= 360? Shouldn't -70 be the same as 290?